interop-testing/src/main/proto/grpc/testing/messages.proto

```diff
@@ -195,4 +195,10 @@ message LoadBalancerStatsResponse {
   map<string, int32> rpcs_by_peer = 1;
   // The number of RPCs that failed to record a remote peer.
   int32 num_failures = 2;
+  message RpcsByPeer {
+    // The number of completed RPCs for each peer.
+    map<string, int32> rpcs_by_peer = 1;
+  }
+  // The number of completed RPCs for each type (UnaryCall or EmptyCall).
+  map<string, RpcsByPeer> rpcs_by_type = 3;
```

Comment:
Let's call this field `rpcs_by_method`. Also updated in https://github.com/grpc/grpc/pull/23436/files

Reply:
Done